R/nested_resampling.R

```diff
@@ -0,0 +1,191 @@
+#' @title Extract Inner Tuning Results
+#'
+#' @description
+#' Extract inner tuning results of nested resampling. Implemented for
+#' [mlr3::ResampleResult] and [mlr3::BenchmarkResult]. The function iterates
+#' over the [AutoTuner] objects and binds the tuning results to a
+#' [data.table::data.table()]. [AutoTuner] must be initialized with
+#' `store_tuning_instance = TRUE` and `resample()` or `benchmark()` must be
+#' called with `store_models = TRUE`.
+#'
+#' @section Data structure:
+#'
+#' The returned data table has the following columns:
+#'
+#' * `experiment` (integer(1))\cr
+#'   Experiment of the benchmark grid.
+#' * `iteration` (integer(1))\cr
+#'   Iteration of the outer resampling.
+#' * One column for each hyperparameter of the search spaces.
+#' * One column for each performance measure.
+#' * `learner_param_vals` (`list()`)\cr
+#'   Tuned and fixed hyperparameter values.
```

Comment:
"fixed" is a bit unclear to me.

Reply:
constant?

Reply:
Parameter values used by the learner. Includes fixed and proposed hyperparameter values.
?